docs/DeveloperGuide.md

```diff
@@ -953,6 +974,40 @@ testers are expected to do more *exploratory* testing.
     5. Other incorrect default view commands to try: `client view`, `client -dv`, `...` <br>
        Expected: Similar to previous.
 
+### Adding an entity
+
+1. Adding an entity while any list of entities is being shown
```

Comment:
A small point is that for project and issue, the prerequisite would include atleast a project being on the list

Reply:
could mention that the issue is added to the list of issues of the project as well